### PR TITLE
Add 5.1 features to administration area

### PR DESCRIPTION
**Pull request by fkaminski**

### CODE REVIEW

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md [hunk 1]
```diff
@@ -40,27 +40,32 @@ Der Infobereich zeigt einige wichtige Links an, die beim Arbeiten mit Contao imm
 
 **Startseite:** Ein Klick auf das Contao-Logo führt zur Startseite des Backends zurück.
 
+**Handbuch:** Beim Klick auf diesen Link wird das Handbuch aufgerufen.
+
+**Favorit speichern:** {{< version-tag "5.1" >}} Beim Klick auf diesen Link kannst du die aktuelle URL im Backend als Favorit Eintrag 
+speichern. Vorhandene Favoriten Links werden im Navigationsbereich aufgeführt.
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md]:
```suggestion
speichern. Vorhandene Favoriten-Links werden im Navigationsbereich aufgeführt.
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md [hunk 2]
```diff
@@ -40,27 +40,32 @@ Der Infobereich zeigt einige wichtige Links an, die beim Arbeiten mit Contao imm
...
+
 **Hinweise:** Beim Klick auf diesen Link öffnet ein Modal und zeigt mögliche Hinweise (z. B. Wartungsmodus) an.
 
-**Debug-Modus:** Beim Klick auf diesen Link wird der [Debug-Modus](../../system/debug-modus/) eingeschaltet bzw. ausgeschaltet.
-{{< version "4.8" >}}
+**Design:** {{< version-tag "5.1" >}} Du kannst zwischen hellen und dunklen Backend Design wählen.
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md]:
```suggestion
**Design:** {{< version-tag "5.1" >}} Du kannst zwischen hellen und dunklen Backend-Design wählen.
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md [hunk 3]
```diff
@@ -40,27 +40,32 @@ Der Infobereich zeigt einige wichtige Links an, die beim Arbeiten mit Contao imm
...
+oder einen Artikel im Backend bearbeitest, wirst du automatisch auf die entsprechende Seite im Frontend weitergeleitet.
 
 **Benutzer:** Beim Klick auf Benutzer werden folgende Links angezeigt:
+  + **Profil:** Dieser Link führt zu den persönlichen Einstellungen deines Benutzerkontos. Du kannst dort z. B. dein Passwort ändern oder 
+  die Sprache wechseln.
 
-**Profil:** Dieser Link führt zu den persönlichen Einstellungen deines Benutzerkontos. Du kannst dort z. B. dein
-Passwort ändern oder die Sprache wechseln.
+  + **Sicherheit:** {{< version-tag "4.6" >}} Über diesen Link kann die [Zwei-Faktor-Authentifizierung](https://de.wikipedia.org/wiki/Zwei-Faktor-Authentisierung) 
+  für dein Backend aktiviert werden.
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.de.md]:
```suggestion
  für die Anmeldung im Backend aktiviert werden.
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md [hunk 4]
```diff
@@ -31,20 +31,30 @@ The info area shows some important links that are needed when working with Conta
 
 **Homepage:** A click on the Contao logo takes you back to the homepage of the back end.
 
+**Manual:** Clicking on this link opens the manual.
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md]:
```suggestion
**Manual:** Clicking this link opens the manual.
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md [hunk 5]
```diff
@@ -31,20 +31,30 @@ The info area shows some important links that are needed when working with Conta
...
+
+**Save Favorite:** {{< version-tag "5.1" >}} By clicking on this link you can save the current backend URL as a favorite entry. Existing
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md]:
```suggestion
**Save Favorite:** {{< version-tag "5.1" >}} By clicking on this link you can save the current back end URL as a favorite entry. Existing 
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md [hunk 6]
```diff
@@ -31,20 +31,30 @@ The info area shows some important links that are needed when working with Conta
 
 **Homepage:** A click on the Contao logo takes you back to the homepage of the back end.
 
+**Manual:** Clicking on this link opens the manual.
+
+**Save Favorite:** {{< version-tag "5.1" >}} By clicking on this link you can save the current backend URL as a favorite entry. Existing 
+favorite links are listed in the navigation area.
+
 **Notes:** Clicking on this link opens a modal and displays possible hints (e.g. maintenance mode).
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md]:
```suggestion
**Notes:** Clicking this link opens a modal and displays possible hints (e.g. maintenance mode).
```

File: docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md [hunk 7]
```diff
@@ -31,20 +31,30 @@ The info area shows some important links that are needed when working with Conta
...
 
-**Debug Mode:** Clicking on this link will switch the [debug mode](/en/system/debug-mode/) on or off.{{< version "4.8" >}}
+**Design:** {{< version-tag "5.1" >}} You can choose between light and dark mode.
+
+**Debug Mode:** {{< version-tag "4.8" >}} Clicking on this link will switch the [debug mode](/en/system/debug-mode/) on or off.
```
Comment (netzarbeiter) [inline docs/manual/administration-area/accessing-and-structure-of-the-backend.en.md]:
```suggestion
**Debug Mode:** {{< version-tag "4.8" >}} Clicking this link will switch the [debug mode](/en/system/debug-mode/) on or off.
```

**Review by netzarbeiter — CHANGES_REQUESTED** [review]
Thank you, Franko

**Comment by fkaminski** [comment]
Thank you

**Review by netzarbeiter — APPROVED** [review]

**Pull request merged**